cmd/orphan-finder/main.go

```diff
@@ -255,7 +255,7 @@ func setup(configFile string) (blog.Logger, core.StorageAuthority, core.Certific
 
 	caConn, err := bgrpc.ClientSetup(conf.OCSPGeneratorService, tlsConfig, clientMetrics, cmd.Clock())
 	cmd.FailOnError(err, "Failed to load credentials and create gRPC connection to CA")
-	cac := bgrpc.NewCertificateAuthorityClient(nil, capb.NewCertificateAuthorityClient(caConn))
+	cac := bgrpc.NewCertificateAuthorityClient(nil, capb.NewOCSPGeneratorClient(caConn))
```

Comment:
Question: why can't we just use `NewOCSPGeneratorClient` here (and above?) the comment in `cmd/ocsp-updater/main.go` seems to imply that we should've eventually been able to do that. Given you've deleted that is it no longer true?

Reply:
Ah, I think I misunderstood the comment I removed. I assumed it was a holdover from the RabbitMQ days. But yeah in theory we should be able to skip the `NewCertificateAuthorityClient` wrapper here. Giving it a shot...

Reply:
Ok, I've pushed a change that does this in orphan-finder, and undoes the commend deletion in ocsp-updater. We can tackle that as a separate change.

Reply:
Sounds good, could you create a ticket for the follow-up work and link it in the comment.